### PR TITLE
add persistent volume for pyrometer

### PR DESCRIPTION
**Pull request by nicolasochem**
Pyrometer saves its message queue on disk. Upon restart, we ensure that
the messages in the queue will be processed.

### CODE REVIEW

**Review by itkach — APPROVED** [review]
lgtm

File: charts/pyrometer/templates/volume.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: "v1"
+kind: PersistentVolumeClaim
+metadata:
+  name: pyrometer-volume
```
Comment (harryttd) [inline charts/pyrometer/templates/volume.yaml:4]: Can you add namespace field?

Comment (nicolasochem) [reply charts/pyrometer/templates/volume.yaml:4]: sure, I was wondering why do we have namespace fields in all our charts?

Comment (harryttd) [reply charts/pyrometer/templates/volume.yaml:4]: It's a normal field in namespace scoped resources. The Helm generated charts add it. I've missed adding it before and it created issues with the resource being deployed in the wrong namespace. I believe the default one.

**Review by harryttd — APPROVED** [review]

File: charts/pyrometer/templates/volume.yaml (hunk 1 above)
Comment (nicolasochem) [reply charts/pyrometer/templates/volume.yaml:4]: But I assume when you use `helm --namespace x` then all resources will go to x. except perhaps the namespace-less control plane resources.

Comment (harryttd) [reply charts/pyrometer/templates/volume.yaml:4]: i know that with `helm template` no namespace is added to the resources. Even if you pass `--namespace`

**Pull request merged**